libs/@guardian/source-react-components-development-kitchen/src/file-input/types.ts

```diff
@@ -0,0 +1,36 @@
+export interface FileInputProps {
+	id?: string;
```

Comment:
This is so nice and clear Sophie - thanks for making it so easy to read! Do we want to be able to pass in `format` so that the button can be styled accordingly? News, opinion etc.

Reply:
I think we should just be able to override the primary colour in the theme to do that! Seems to be the pattern in source-react-components anyway...

Reply:
Okay cool - if at all possible, could we jump on a call just so you can show me how I'd do that for my callout PR?